hack/remove-members.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+readonly REPO_ROOT="$(git rev-parse --show-toplevel)"
+readonly CONFIG_PATH="$REPO_ROOT/config"
+readonly DRYRUN="${DRYRUN:-true}"
```

Comment:
Can we make this a flag?

Reply:
Yeah, this was me being kind of lazy to get it done lol